### DISCUSSION TITLE
Next release?

### DISCUSSION THREAD

**Discussion by jhm-ciberman** [HandyOrg/HandyControl#1189] [General]
Hello! 
First, thanks again for creating a wonderful UI library. We are a tiny two-person software business currently developing a Cashing and accounting system for Automotor registration business. We are in closed beta right now and we are getting feedback from real users. They love the UI. And as a developer I'm in love with HandyControl. 

I would like to know if there is a next release planed as we a are interested in some specific issues that are not currently in the public NuGet release. (We use the forked version by @ghost1372)

These are the specific issues we are interested, in order of importance for our application: 
- [ ] https://github.com/HandyOrg/HandyControl/issues/1144
- [ ] https://github.com/HandyOrg/HandyControl/issues/998
- [x] https://github.com/HandyOrg/HandyControl/issues/1142
- [ ] https://github.com/HandyOrg/HandyControl/issues/1073
- [x] https://github.com/HandyOrg/HandyControl/issues/1065

I haven't forked the library on my own, but if you need help with something, let me know. 
Also, If you ever need any other translation to Spanish, simply tag me and I will do it :) 

Thanks for everything! Love <3 HandyControls.

**Comment by NaBian** [comment]
I've been so busy these two months! 😭

**Reply by jhm-ciberman** [reply]
Oh, no problem, mate! I really love the library and the amazing effort you are doing with it. There is no hurry. None of these features are absolutely required for our app. Thank you for everything! <3

**Comment by xianyunleo** [comment]
Me too ,thank you
